frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -30,13 +38,23 @@ const StyledLink = styled(Link)(({ theme }) => ({
 
 const GridContainer = styled(Box)(() => ({
     width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+}));
+
+const ScrollableContent = styled(Box)(({ theme }) => ({
+    width: '100%',
+    maxHeight: '70vh',
+    overflowY: 'auto',
+    padding: theme.spacing(0, 0, 1, 0),
 }));
 
 const GridSection = styled(Box)(({ theme }) => ({
     display: 'grid',
     gridTemplateColumns: 'repeat(2, 1fr)',
     gap: theme.spacing(1.5),
     padding: theme.spacing(0, 2),
+    marginBottom: theme.spacing(3),
```

Comment:
The GridSection component is housed within the GridContainer, which is a flex container. Wouldn't it be better to use `gap: theme.spacing(3)` to GridSection instead? That way, the children (grid containers) don't need to worry about how they're positioned next to their siblings. The container already covers layout direction, so I think it'd be useful to move that up there too (colocates the responsibility)

```suggestion
```

Reply:
Or is that not right because you also have a nested box? In that case, you can add the same gap to that one. Or: how about just applying spacing when there's more siblings?

```tsx
'& + &': { marginTop: theme.spacing(3) }
```
 will only add the margin between the items if there's more than one in a row 💁🏼

Reply:
Not exactly, then all the cards will have this spacing.